wurst/systems/meatsystem/EatRawMeat.wurst

```diff
@@ -70,10 +70,7 @@ class EatRawMeatInstance
         doPeriodicallyTimed(ANIMATION_PERIOD, CHANNEL_TIME) cb ->
             if this.isChanneling and caster.isAlive()
                 caster.setHP(caster.getHP() + HP_PER_SECOND * ANIMATION_PERIOD)
-            else
-                cb.stop()
-
-            if cb.isLast() or not (this.isChanneling and caster.isAlive())
+            else if cb.isLast() or not (this.isChanneling and caster.isAlive())
```

Comment:
This would be the same as an `else` branch without a condition.

Reply:
Oops. Looks like i rushed that a bit. Fixed